docs/word/word-add-ins-events.md

```diff
@@ -0,0 +1,199 @@
+| `onCommentDeselected` | Occurs when a comment is deselected.<br><br>Event data object:<br>[CommentEventArgs](/javascript/api/word/word.commenteventargs) | <ul><li>[**Body**](/javascript/api/word/word.body?view=word-js-preview&preserve-view=true#word-word-body-oncommentdeselected-member)</li><li>[**ContentControl**](/javascript/api/word/word.contentcontrol?view=word-js-preview&preserve-view=true#word-word-contentcontrol-oncommentdeselected-member)</li><li>[**Paragraph**](/javascript/api/word/word.paragraph#word-word-paragraph-oncommentdeselected-member)</li><li>[**Range**](/javascript/api/word/word.range#word-word-range-oncommentdeselected-member)</li></ul> |
+| `onCommentSelected` | Occurs when a comment is selected.<br><br>Event data object:<br>[CommentEventArgs](/javascript/api/word/word.commenteventargs) | <ul><li>[**Body**](/javascript/api/word/word.body?view=word-js-preview&preserve-view=true#word-word-body-oncommentselected-member)</li><li>[**ContentControl**](/javascript/api/word/word.contentcontrol?view=word-js-preview&preserve-view=true#word-word-contentcontrol-oncommentselected-member)</li><li>[**Paragraph**](/javascript/api/word/word.paragraph#word-word-paragraph-oncommentselected-member)</li><li>[**Range**](/javascript/api/word/word.range#word-word-range-oncommentselected-member)</li></ul> |
+
+### Event triggers
+
+Events within a Word document can be triggered by:
+
+- User interaction via the Word user interface (UI) that changes the document.
+- Office Add-in (JavaScript) code that changes the document.
+- VBA add-in (macro) code that changes the document.
+
+Any change that complies with default behavior of Word will trigger the corresponding events in a document.
+
+### Lifecycle of an event handler
+
+An event handler is created when an add-in registers the event handler. It's destroyed when the add-in deregisters the event handler or when the add-in is refreshed, reloaded, or closed. Event handlers don't persist as part of the Word file, or across sessions with Word on the web.
+
+> [!CAUTION]
+> When an object to which events are registered is deleted (e.g., a content control with an `onDataChanged` event registered), the event handler no longer triggers but remains in memory until the add-in or Word session refreshes or closes.
+
+### Events and coauthoring
+
+With coauthoring, multiple people can work together and edit the same Word document simultaneously. For events that can be triggered by a coauthor, such as `onParagraphChanged`, the corresponding **Event** object will contain a **source** property that indicates whether the event was triggered locally by the current user (`event.source = Local`) or was triggered by the remote coauthor (`event.source = Remote`).
```

Comment:
Should "triggered by coauthoring" be a table column entry?

Reply:
Have you definitely validated this works? My recollection from Excel is that coauth changes do not trigger events. Your add on line 50 does not include coauth changes - should it?